src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Runtime.pkgproj

```diff
@@ -16,4 +14,12 @@
   <Import Condition="'$(ExcludeHostAssets)' != 'true'" Project="..\..\Microsoft.NETCore.DotNetHostPolicy\Microsoft.NETCore.DotNetHostPolicy.props" />
   <Import Condition="'$(ExcludeHostAssets)' != 'true'" Project="..\..\Microsoft.NETCore.DotNetHostResolver\Microsoft.NETCore.DotNetHostResolver.props" />
 
+  <!-- Keep this isolated to desktop mono for now -->
+  <Target Name="MobileGetBuildRidSpecificPackageProps"
+          Condition="'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' == ''"
```

Comment:
Nit: change the condition to `'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' != 'true'`